### ISSUE TITLE
SnackBar timers don't seem to follow accessibleNavigation rule

### ISSUE THREAD

**Issue by Piinks** [flutter/flutter#64832]
[`MediaQuery.accessibleNavigation`](https://api.flutter.dev/flutter/widgets/MediaQueryData/accessibleNavigation.html) states
>When this setting is true, features such as timeouts should be disabled or have minimum durations increased.

This seems to be partially followed by `SnackBar`s.

When `accessibleNavigation` is true, and a `SnackBar` has an `action`, the timeout is disabled, allowing the user to dismiss the `SnackBar` manually using the `action`.

When `accessibleNavigation` is true, and a `SnackBar` does not have an `action`, the timeout runs as usual. It would seem by the documentation above that in this case the timer should be longer. We should confirm this and update the timers for `accessibleNavigation`.

BTW - the timers are currently set in `ScaffoldState`
https://github.com/flutter/flutter/pull/64101 will move this from `ScaffoldState` to `ScaffoldMessengerState`

**Comment by jonahwilliams** [comment]
I believe the current behavior is correct: see go/snackbar-a11y-spec

**Comment by Piinks** [comment]
Thanks @jonahwilliams!
FYI @hansmuller 
I'll close this then. :)
